packages/suite/src/components/wallet/AccountAnnouncement/CoinjoinContextMessage.tsx

```diff
@@ -2,15 +2,19 @@ import React from 'react';
 import { NotificationCard } from '@suite-components';
 import { Link } from '@trezor/components';
 import { useSelector } from '@suite-hooks';
-import { Context, selectContextMessageContent } from '@suite-reducers/messageSystemReducer';
+import { selectContextMessageContent, Context } from '@suite-common/message-system';
+import { selectLanguage } from '@suite-reducers/suiteReducer';
 import { Account } from '@wallet-types';
 
 type CoinjoinContextMessageProps = {
     account?: Account;
 };
 
 export const CoinjoinContextMessage = ({ account }: CoinjoinContextMessageProps) => {
-    const message = useSelector(state => selectContextMessageContent(state, Context.coinjoin));
+    const language = useSelector(selectLanguage);
+    const message = useSelector(state =>
+        selectContextMessageContent(state, Context.coinjoin, language),
```

Comment:
Yes, that's how it was, but I rewrote it because I believe it's easier to reuse it with language as param in suite-native. Otherwise we need to define it as extraDependencies, because `selectLanguage` is not shared. Of course both ways are possible.

Reply:
This is okay, better than messing with extraDeps.

Reply:
Okay, got it. No problem